docs/develop/getting-started/quick_start.md

```diff
@@ -25,17 +25,27 @@ The easiest way to install WasmEdge is to run the following command. You should
 curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | sudo bash -s -- -p /usr/local
 ```
 
-> If you do not have root or `sudo` rights, use the following line to install WasmEdge in your `$HOME/.wasmedge` directory: `curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash`
+> If you do not have root or `sudo` rights, use the following line to install WasmEdge in your `$HOME/.wasmedge` directory:
+
+```bash
+ `curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash`
+```
 
 
 ## Run a standalone Wasm app
 
 The Hello world example is a standalone Rust application that can be executed by the [WasmEdge CLI](../build-and-run/cli). Its source code and build instructions are available [here](https://github.com/second-state/rust-examples/tree/main/hello).
 
-[Download the hello.wasm file here](/files/hello.wasm)
+Download the hello.wasm file [here](/files/hello.wasm), or run the following command:
```

Comment:
This are hashed files which are bundled by webpack the original files exist at the following path: https://wasmedge.org/docs/files/hello.wasm

![image](https://user-images.githubusercontent.com/84327775/235051112-892d7a51-b0ff-45ec-931a-6e7ec16b2a71.png)